packages/create-plugin/README.md

```diff
@@ -99,6 +99,18 @@ For more information see [here](https://grafana.com/developers/plugin-tools/migr
 
 You can read more about customizing or extending the basic configuration [here](https://grafana.com/developers/plugin-tools/create-a-plugin/extend-a-plugin/extend-configurations)
 
+
```

Comment:
Perhaps a title for this section is required? As is it looks like this command is part of the `Customizing or extending the basic configs` section

Reply:
Also, It'd be best to add an additional description of what is this command doing or why you would like to run it

Reply:
Yeah, it should be under its own section (e.g. ~ "Add provisioning to your existing plugin").